### PR TITLE
[feature] piv: manage modification_limitation

### PR DESCRIPTION
**Pull request by patochectp**
ref: ND-1114

### CODE REVIEW

**Review by pbougue — APPROVED** [review]
:smile: :+1: 

Is it possible to add a test?

File: kirin/piv/model_maker.py [hunk 1]
```diff
@@ -79,8 +79,13 @@
     TripEffect.UNDEFINED: maxint,
 }
 
-MANAGED_EVENTS = ["RETARD", "SUPPRESSION", "MODIFICATION_DESSERTE_SUPPRIMEE"]
-MANAGED_STOP_EVENTS = ["RETARD_OBSERVE", "RETARD_PROJETE", "NORMAL", "SUPPRESSION_PARTIELLE"]
+MANAGED_EVENTS = ["RETARD", "SUPPRESSION", "MODIFICATION_DESSERTE_SUPPRIMEE", "MODIFICATION_DESSERTE_SUPPRIMEE"]
```
Comment (pbougue) [inline kirin/piv/model_maker.py:82]:
```suggestion
MANAGED_EVENTS = ["RETARD", "SUPPRESSION", "MODIFICATION_DESSERTE_SUPPRIMEE", "MODIFICATION_LIMITATION"]
```

File: tests/fixtures/piv/stomp_20201022_23186_modification_limitation.json [hunk 2]
```diff
@@ -0,0 +1,723 @@
...
+            },
+            "evenement":[
+               {
+                  "rangDesserte":[
+                     2,
+                     3,
+                     4
+                  ],
+                  "rangEvt":3,
+                  "type":"MODIFICATION_DESSERTE_SUPPRIMEE",
```
Comment (pbougue) [inline tests/fixtures/piv/stomp_20201022_23186_modification_limitation.json:612]:
```suggestion
                  "type":"MODIFICATION_LIMITATION",
```

File: tests/integration/piv_test.py [hunk 3]
```diff
@@ -405,3 +405,56 @@ def test_piv_partial_removal(mock_rabbitmq):
...
+            assert len(db_trip_partial_removed.stop_time_updates) == 17
+
+            # the first stop have not been changed
+            first_st = db_trip_partial_removed.stop_time_updates[0]
+            assert first_st.stop_id == "stop_point:PIV:85010231:Train"
+            assert first_st.arrival_status == "none"
+            assert first_st.departure_status == "none"
+            assert first_st.message is None
+
+            # the second stop have been departure marked as deleted
```
Comment (pbougue) [inline tests/integration/piv_test.py:447]: minor
```suggestion
            # the second stop's departure have been marked as deleted
```

**Review by pbougue — CHANGES_REQUESTED** [review]
Side-note: after #345 we'll have shorter train 23187 to copy/paste !

**Review by woshilapin — COMMENTED** [review]
Do we need a new fixture for this case or can it be merged into one of the existing fixture?

File: kirin/piv/model_maker.py [hunk 4]
```diff
@@ -79,8 +79,13 @@
...
+MANAGED_STOP_EVENTS = [
```
[the remark this answers is not in the document]
Comment (pbougue) [reply kirin/piv/model_maker.py:83]: It had modification in a previous blacked version of the code... And it looks like black doesn't have one OK way for a given code :shrug:

**Review by pbougue — APPROVED** [review]

**Review by woshilapin — APPROVED** [review]

**Pull request merged**